.github/workflows/agw-build-publish-container.yml

```diff
@@ -268,4 +268,4 @@ jobs:
           SLACK_AVATAR: ":boom:"
         uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # pin@2.1.0
         with:
-          args: "Containerized LTE integration test failed on [${{ github.sha }}](${{github.event.repository.owner.html_url}}/magma/commit/${{ github.sha }}): ${{ steps.commit.outputs.title}}"
```

Comment:
Here we remove the reference to a step output, but there doesn't seem to be a corresponding step. I'm puzzled how this worked before.

Reply:
It didn't, for most of these workflows actually. If you take a look at the `#ci` Slack channel, no failure message (except from FEG integ test and some Bazel workflows) contains anything after `:`.